SofaKernel/framework/sofa/core/behavior/BaseMechanicalState.h

```diff
@@ -226,6 +226,9 @@ class SOFA_CORE_API BaseMechanicalState : public virtual BaseState
     /// build the jacobian of the constraint in a baseMatrix
     virtual void getConstraintJacobian(const ExecParams* params, sofa::defaulttype::BaseMatrix* J,unsigned int & off) = 0;
 
+    /// fill the jacobian matrix (of the constraints) with identity blocks on the provided list of nodes(dofs)
```

Comment:
For the usage I have in mind, it would be more useful to start building the sparse Jacobians from ForceFields/Constraints. I am not sure of the interest of starting from a full Identity.
Starting from a selection matrix (identity with 0s) corresponding to the dofs where the ForceFields/Constraints are contributing allows to have the as parse as possible Jacobians, possibly null for branches where there is no ForceFields/Constraints.
Note building such a selection matrix is actually managed by `BaseForceField::updateForceMask`.